db/errors/index.js

```diff
@@ -1,5 +1,5 @@
 exports.psqlErrorHandler = (err, request, response, next) => {
-  console.log(err);
+  // console.log(err);
   if (["42703", "42601"].includes(err.code)) {
```

Comment:
What are these error codes for? At this stage I'd expect to see a handler for 22P02 but not much more.